test/utils/utils.test.ts

```diff
@@ -0,0 +1,28 @@
+const { earnDataParser } = require( '../../lib/utils/utils.js')
+const { fixture } = require('./utilsFixtures')
+
+const assert = require('chai').assert
+
+describe('earnDataParser', () => {
+  for (const testCase of fixture.earnDataParser) {
+    it(testCase.testDescription, () => {
+      // Arrange
+      const inputData = testCase.inputData
+      const expectedType = testCase.outputType
+      const expectedOutput = testCase.expectedOutput
+      // Act
+      // Assert
```

Comment:
The `Act` and `Assert` looks different based on the `expectedType`. I would move the `Act` comment above line 20, and the `Assert` above line 21.